qemu/tests/cfg/hotplug_unplug_with_viommu.cfg

```diff
@@ -0,0 +1,21 @@
+- hotplug_unplug_with_viommu:
+    virt_test_type = qemu
+    type = nic_hotplug
+    required_qemu= [7.0.0,)
+    only q35
+    only Linux
+    only virtio_net
+    login_timeout = 360
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    repeat_times = 1
+    virtio_iommu = yes
+    pci_model = virtio-net-pci
+    netdev_extra_params_hotplug_nic1 = ",vhost=on"
+    nic_extra_params_hotplug_nic1 = ",iommu_platform=on"
+    variants:
+        - direct_plug:
+              virtio_iommu_direct_plug = yes
+        - non_direct_plug:
+              virtio_iommu_direct_plug = no
```

Comment:
```suggestion
```

Reply:
This line will be removed after next update.